### PR TITLE
fix(playht): add sample_rate parameter to JSON payload

### PR DESCRIPTION
**Pull request by imsakg**
Ensure the sample_rate is included in the JSON data for text-to-speech requests, aligning with other audio options.

### CODE REVIEW

**Comment by imsakg** [comment]
It was causing the audio at 2x speed.

**Review by theomonnom — APPROVED** [review]
Otherwise lgtm!

File: livekit-plugins/livekit-plugins-playht/livekit/plugins/playht/tts.py [hunk 1]
```diff
@@ -87,13 +87,14 @@ def __init__(
         user_id: str | None = None,
         base_url: str | None = None,
         encoding: _TTSEncoding = "wav",
+        sample_rate: TTSSampleRate = PLAYHT_TTS_SAMPLE_RATE,
```
[the remark this answers is not in the document]
Comment (imsakg) [reply livekit-plugins/livekit-plugins-playht/livekit/plugins/playht/tts.py:90]: There is lots of different sample rates formats such as 8000,22100,24000,32000,44100,48000 and I believe users could write wrong numbers by mistake. Also they need to check to know which rates are supported, with this design we eliminate both problems I think. What do you think?

[the remark this answers is not in the document]
Comment (theomonnom) [reply livekit-plugins/livekit-plugins-playht/livekit/plugins/playht/tts.py:90]: this is fair, but we're already exposing it as a int on the other TTSs. So my vote would be to be consistent here.

**Review by theomonnom — APPROVED** [review]
Thanks lgtm!

**Pull request merged**